src/main/java/graphql/schema/GraphQLDirective.java

```diff
@@ -342,6 +358,19 @@ public Builder onField(boolean onField) {
             return this;
         }
 
+        /**
+         * @param onVariableDefinition onVariableDefinition
+         *
+         * @return this builder
+         *
+         * @deprecated Use {@code graphql.schema.GraphQLDirective.Builder#validLocations(DirectiveLocation...)}
```

Comment:
lets not add new methods which are immediately deprecated.

Reply:
copy that.